src/tests/eina/eina_suite.c

```diff
@@ -53,7 +53,7 @@ static const Efl_Test_Case etc[] = {
    { "Module", eina_test_module },
    { "Convert", eina_test_convert },
    { "Rbtree", eina_test_rbtree },
-   { "File", eina_test_file },
+   // { "File", eina_test_file },
```

Comment:
Oh, I think this was for testing, right?

Reply:
Oops, this shouldn't have gone through! I'll redo the commit and force push before merging

Reply:
we can just add a new commit, and we can squash it in the end. Do as you prefer.

Reply:
Fair enough. Will do.